clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -195,7 +195,10 @@ public class ConsumerConfig extends AbstractConfig {
             "Records are fetched in batches by the consumer, and if the first record batch in the first non-empty partition of the fetch is larger than " +
             "this value, the record batch will still be returned to ensure that the consumer can make progress. As such, this is not a absolute maximum. " +
             "The maximum record batch size accepted by the broker is defined via <code>message.max.bytes</code> (broker config) or " +
-            "<code>max.message.bytes</code> (topic config). Note that the consumer performs multiple fetches in parallel.";
+            "<code>max.message.bytes</code> (topic config). A fetch request consists of many partitions, and there is another setting that controls how much " +
+            "data is returned for each partition in a fetch request - see <code>max.partition.fetch.bytes</code>. Note that there is a current limitation when " +
+            "performing remote reads from tiered storage (KIP-405) - only one partition out of the fetch request is fetched from the remote store (KAFKA-14915). " +
+            "Note also that the consumer performs multiple fetches in parallel.";
```

Comment:
> Note also that the consumer performs multiple fetches in parallel.

This statement is unclear to me. Does this refer to: 

1. Consumer reads from multiple brokers as the partitions gets spread out?
2. Multiple consumers with different client-id reading from a broker?

Reply:
I assume it means a single consumer will send multiple fetch requests. Note @kamalcph I didn't add this - it was here